### PR TITLE
Update virtual-kubelet dependency to v1.5.0

### PR DESCRIPTION
**Pull request by cwdsuzhou**
When I update tensile-kube dependency to vk v1.5.0, I find node-cli cannot work well with node-cli.
This PR updates node-cli to vk v1.5.0

### CODE REVIEW

File: internal/commands/root/root.go [hunk 1]
```diff
@@ -166,37 +165,34 @@ func runRootCommandWithProviderAndClient(ctx context.Context, pInit provider.Ini
...
-			if err != nil {
-				return err
-			}
-			log.G(ctx).Debug("created new node")
-			return nil
-		}),
-	)
+
+	var opts []node.NodeControllerOpt
+	if c.EnableNodeLease {
```
Comment (pires) [inline internal/commands/root/root.go:175]: Let's default this to true, since we're at it.

**Review by pires — COMMENTED** [review]
LGTM

File: internal/commands/root/root.go (hunk 1 above)
Comment (cwdsuzhou) [reply internal/commands/root/root.go]: done

File: opts/opts.go [hunk 2]
```diff
@@ -160,7 +165,10 @@ func setDefaults(o *Opts) {
 	o.KubeClusterDomain = DefaultKubeClusterDomain
 	o.StreamIdleTimeout = DefaultStreamIdleTimeout
 	o.StreamCreationTimeout = DefaultStreamCreationTimeout
-	o.RateLimiter = workqueue.DefaultControllerRateLimiter()
+	o.EnableNodeLease = true
```
Comment (cwdsuzhou) [inline opts/opts.go:168]: set default true here

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: opts/opts.go (hunk 2 above)
Comment (pires) [reply opts/opts.go:168]: Thanks

File: internal/commands/root/root.go (hunk 1 above)
Comment (pires) [reply internal/commands/root/root.go]: Thanks

**Pull request merged**